### ISSUE TITLE
Start removing hard-coded color values

### ISSUE THREAD

**Issue by itskarelleh** [itskarelleh/taino-language-app#34]
Here are the following things that need to be done to complete this issue:
- [ ] All components should no longer have any hardcoded hex codes
- [ ] All components that are using color attributes should be replaced using Colors.tsx
- [ ] Remove any additional styles in components that might be overriding the styles

Example:
```tsx
  buttonText: {
    color: "#475467", // this is not good ❌
    fontFamily: "Inter", 
    fontStyle: "normal",
    fontWeight: "700",
    lineHeight: 1,
  }

buttonText: {
    color: Colors.onPrimary.highEmphasis, // this is good ✅
    fontFamily: "Inter",
    fontStyle: "normal",
    fontWeight: "700",
    lineHeight: 1,
  }
```

**Comment by itskarelleh** [comment]
Solved with #41

**Comment by itskarelleh** [comment]
Solved with #41
